src/callbacks/callbacks.jl

```diff
@@ -524,3 +526,16 @@ function gc_func(integrator)
     )
     return nothing
 end
+
+
+function update_surface_temp!(integrator)
+    d = 10 # ocean mixed layer depth [m]
+    ρₒ = 1020 # ocean density [kg / m³]
+    cₚₒ = 4184 # ocean heat capacity [J/(kg * C°)]
```

Comment:
```suggestion
    cp_ocean = 4184 # ocean heat capacity [J/(kg * K)]
```
(which is the same, I am just changing it to SI unit)